pebble/concurrent/process.py

```diff
@@ -43,25 +43,28 @@ def process(*args, **kwargs):
     The timeout parameter will set a maximum execution time
     for the decorated function. If the execution exceeds the timeout,
     the process will be stopped and the Future will raise TimeoutError.
-
+    The name parameter will set the process name.
     """
     timeout = kwargs.get('timeout')
+    name = kwargs.get('name')
```

Comment:
The `dict.get()` method already returns `None` if there's no occurrence. So you can safely propagate the `name` value to the launcher as it will correctly handle it.